docs/01-installing-gradlerio.md

```diff
@@ -6,22 +6,36 @@ This is the guide for installing GradleRIO for your project. If you're looking t
 If you don't have Java installed on your system, you can download it from [here](https://java.com/en/download/). Any version 8 or higher will work just fine.
 
 ## Step 2. Download Quickstart
-Download the Quickstart from [here](../Quickstart.zip). Extract the Quickstart and choose the directory for your language (either java or cpp), and copy it to your project directory.
+Download the Quickstart from [here](../Quickstart.zip). Extract the Quickstart files to a temporary location on your disk.
+
+## Step 3a.  Option 1 - Create a new project
+The quickstart files will give you the basis for a new robot project, which you can simply
+use to begin creating a robot.  To do this, simply copy the entire directory for your language
+(either java or cpp) to the location of your choice.  For Java, rename the team0000 directory to something
+more to your liking.  For cpp, you will likely want to rename 'MyRobot.cpp' and 'MyHeader.h' to
+something more to your liking.
+
+## Step 3b.  Option 2 - Update an existing project
+If you have an existing project, you can copy in most of the files from the quickstart directory.  For Java,
+you will not need the 'team0000' directory.  For cpp, you will not need the 'MyRobot.cpp' or
```

Comment:
I feel like this wording could be better. Maybe extract the quickstart, delete the contents of `src/main/java` or `src/cpp`, and copy in the new files under `src/main/java` or `src/cpp` (a lot of teams just chuck it in `src`).